Web3Swift/StringScalar/TrimmedPrefixString.swift

```diff
@@ -38,15 +40,10 @@ public final class TrimmedPrefixString: StringScalar {
     public func value() throws -> String {
         let string = try self.string.value()
         let prefix = try self.prefix.value()
-        if string.hasPrefix(prefix) {
-            return String(
-                string.dropFirst(
-                    Int(prefix.count)
-                )
-            )
-        } else {
-            return string
-        }
+        let pattern = "^(\(prefix)){1,}"
+        guard let range = string.range(of: pattern, options: [.regularExpression]) else { return string }
+        let trimmed = String(string[range.upperBound...])
+        return (!trimmed.isEmpty) ? trimmed : prefix
```

Comment:
Please replace ternary conditional operator with `if else`. Don't use `if not else`